processccd.rst

```diff
@@ -1,131 +1,100 @@
-Characterization', which includes for our purposes: object detection
-(very commonly done by Source Extractor), repairing of cosmic ray
-defects, measuring and subtracting of sky background, and then finally
-measuring bright sources and using this to estimate background and PSF
-of an exposure (which is often done currently by astronomers using the
-PSFex code).
-
-The last primary grouping of taks is what we will call 'Image
-Calibration', which measures faint sources, does the astrometry by
-fitting an improved WCS to the image (often done currently by
-astronomers using by using the SCAMP and SWARP codes, 'pinning' the
-image on the positions of known stars), and figures out the
-photometric zero-point for the image.
-
-
-How to use the ProcessCcd Task
-++++++++++++++++++++++++++++++
+	- detect sources, usually at high S/N
+	- estimate the background, which is subtracted from the image and returned as field "background"
+	- estimate a PSF model, which is added to the exposure
+	- interpolate over defects and cosmic rays, updating the image, variance and mask planes
+    
+ 
+-	``calibrate`` - target=CalibrateTask - Task to perform astrometric and photometric calibration, the steps are to:
 
-This task is primarily designed to be run from the command line.  The
-main method is 'run', which takes a single (butler) data reference for the
-raw input data (i.e. the directory in which all the raw data files are
-stored, and where all the processed data files are to be written).
+	- refine the WCS in the exposure
+	- refine the Calib photometric calibration object in the exposure
+	- detect sources, usually at low S/N
+ 
 
-..
- Preparing the data for ProcessCcd 
- ---------------------------------
+Entrypoint
+==========
 
+- ``lsst.pipe.tasks.processCcd.ProcessCcdTask.run`` 
+  
 
+Butler Inputs
+=============
 
+The main method, ``run``, takes a single butler data reference for the raw input data.
 
-Configuration parameters
-------------------------
+Examples
+========
 
-The Task has several Configurable Fields (functions of the pexConfig class):
+The following commands will process all raw data in obs_test's data repository. Note: be sure to specify an ``--output`` that does not already exist::
 
-  - isr, which has as its target IsrTask
+  setup obs_test
+  setup pipe_tasks
+  processCcd.py $OBS_TEST_DIR/data/input --output processCcdOut --id
 
-  - charImage, which has as its target CharacterizeImageTask
+The data is read from the small repository in the ``obs_test`` package and written to: ``./processCcdOut`` (or whatever output you specified). Specifying ``--id`` with no values processes all data. Add the option ``--help`` to see more options.
 
-  - calibrate, which has as its target CalibrateTask
 
-    
 Debugging
-----------
+=========
 
 ProcessCcdTask has no debug output, but its several subtasks do.
 
-Brief Outline with Subtask Code Names
-++++++++++++++++++++++++++++++++++++++
+ArgParse
```

Comment:
I feel like this section itself should have a man page-style command line signature and describe what the command line arguments are.